go-controller/pkg/node/gateway_udn.go

```diff
@@ -659,23 +685,35 @@ func (udng *UserDefinedNetworkGateway) getV6MasqueradeIP() (*net.IPNet, error) {
 // 2000:   from all to 169.254.0.12 lookup 1007
 // 2000:   from all fwmark 0x1002 lookup 1009
 // 2000:   from all to 169.254.0.14 lookup 1009
+// If isPodNetworkAdvertised is set to true then we update IP rules as below
```

Comment:
nit: maybe s/update/replace ?

Reply:
Are you sure it is actually replacing the rules?

Reply:
Ok, I see that you are deleting in `updateUDNVRFIPRule`.

So I would jsut preface the previous example with 
`If the network is not advertise, we set rules...`

And this example with
`If the network is advertised we set rules...`